README.md

```diff
@@ -189,6 +198,9 @@ fmt.Println(ntw.IntegerToDutch(42))
 fmt.Println(ntw.IntegerToTurkish(42))
 // Outputs: kırk iki
 
+fmt.Println(ntw.IntegerToPortuguesePT(42))
+// Outputs: kırk iki
```

Comment:
invalid output ?

Reply:
no, i just forgot to alter it. It outputs: quarenta e dois